distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text.enso

```diff
@@ -149,3 +149,18 @@ type Text
              "aaa".not_empty
     not_empty : Boolean
     not_empty self = self.is_empty.not
+
+    ## ADVANCED
+       PRIVATE
+       UNSTABLE
+
+       Conversion to Text that overrides the default `to_text` behavior.
+    to_text : Text
+    to_text self = Text.from self
+
+## ADVANCED
+   PRIVATE
+   UNSTABLE
+
+   Conversion from Text.
+Text.from (that:Text) = that
```

Comment:
nitpick:

I think `PRIVATE` supersedes `ADVANCED`. We do not have a good guideline, but how I think it's usually used is that `ADVANCED` is used whenever a functionality is meant for user's usage but it is more advanced, only for power users and needs extra care.

`PRIVATE` is usually meaning 'implementation detail, do not rely on it as it may change whenever'.

Then, `UNSTABLE` is rather superseded too as it means 'the API may change in the future' (or now it is more 'this API is _likely_ to change'), so if something is `PRIVATE`, `UNSTABLE` seems rather redundant.

Reply:
Also btw. why do we need `Text.from (that : Text)`? It is a bit weird to see such an identity conversion. Is it _ever_ possible for `T.from (that : T)` to be different from identity? Should that even be allowed? It seems slightly suspicious.